ToNote/Controls/NotePanel.cs

```diff
@@ -129,6 +135,24 @@ public ICommand SaveContentsToFilesCommand
                File.WriteAllText(metadataFileName, serializedNote);
            })));
 
+        public ICommand AddTodoNoteCommand
+        {
+            get { return (ICommand)GetValue(AddTodoNoteCommandProperty); }
+            set { SetValue(AddTodoNoteCommandProperty, value); }
+        }
+
+        public static readonly DependencyProperty AddTodoNoteCommandProperty = DependencyProperty.Register("AddTodoNoteCommand",
+           typeof(ICommand), typeof(NotePanel), new FrameworkPropertyMetadata(new RelayCommand<NotePanel>((panel) =>
+           {               
+               if (panel.lastFocused != null)
```

Comment:
Last focused should track both text boxes and todo controls, so that the user can create a new todo after a selected control. To achieve that TodoControl and ExtendedRichTextBox should probably share a common interface.